dsl_parser/tests/test_inputs.py

```diff
@@ -405,3 +406,25 @@ def test_input_in_outputs(self):
         prepared = prepare_deployment_plan(self.parse(yaml))
         outputs = prepared.outputs
         self.assertEqual(8080, outputs['a']['value'])
+
+    def test_missing_input_exception(self):
+        yaml = """
+node_types:
+  type:
+    interfaces:
+      interface:
+        op:
+          implementation: plugin.mapping
+          inputs:
+            some_input:
+              type: string
+node_templates:
+  node:
+    type: type
+plugins:
+  plugin:
+    install: false
+    executor: central_deployment_agent
+"""
+        self._assert_dsl_parsing_exception_error_code(
```

Comment:
The `107` error code indicates missing properties in general.
So it would be more precise to (also) check that specific expected inputs are missing (in this test's case, `some_input`), in line with the proposed change in `validate_missing_inputs` that displays all the missing inputs together.